cloudformation/template.yml

```diff
@@ -298,13 +304,37 @@ Resources:
           ComputeEnvironment: !Ref BatchComputeEnvironment
           Order: 1
 
-  JobDefinition:
+  CustomJobDefinition:
+    Type: AWS::Batch::JobDefinition
+    Condition: UseCustomDockerImage
+    Properties:
+      Type: Container
+      JobDefinitionName: !Join ['', [!Ref Slug, 'RasterVision', !Ref Environment, 'CustomJobDefinition']]
+      ContainerProperties:
+        Image: !Sub "${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/${RepositoryName}:${ImageTag}"
```

Comment:
Unfortunately, the `Repository` resource [does not expose an attribute corresponding to the URI of the repo](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-repository.html#aws-resource-ecr-repository-returnvalues), meaning that we have to construct the URI by hand here.

Reply:
Note that `ImageName` isn't required for this URI string, since ECR uses the repository name to refer to the image.